### PR TITLE
Fix several MAPINFO tokens

### PR DESCRIPTION
**Pull request by Ch0wW**
This PR fixes many problems reported from issue #358 .

It currently does the following:
- Creates `compat_nopassover` flag.
- Fix "SKY1/SKY2" "scrollspeed" argument being truly optional.
- Fix the "key" and "episode" tokens not being properly parsed.
- Create a fallback for the name if the "lookup" doesn't work.
- Adds "enterpic", "interpic", "exitpic" dummy tokens.

### CODE REVIEW

File: common/g_level.cpp [hunk 1]
```diff
@@ -3355,9 +3376,16 @@ ClusterInfos& getClusterInfos()
 
 
 // P_AllowDropOff()
-bool P_AllowDropOff()
+const bool P_AllowDropOff()
```
Comment (leximayfield) [inline common/g_level.cpp]: You shouldn't return a `const` value type - the `const` is useless.

File: common/g_level.cpp [hunk 2]
```diff
@@ -3355,9 +3376,16 @@ ClusterInfos& getClusterInfos()
...
 {
 	return level.flags & LEVEL_COMPAT_DROPOFF || co_allowdropoff;
 }
 
-VERSION_CONTROL (g_level_cpp, "$Id$")
+const bool P_AllowPassover()
```
Comment (leximayfield) [inline common/g_level.cpp]: You shouldn't return a `const` value type - the `const` is useless.

File: common/g_level.cpp [hunk 3]
```diff
@@ -3355,9 +3376,16 @@ ClusterInfos& getClusterInfos()
...
+{
+	if (level.flags & LEVEL_COMPAT_NOPASSOVER)
+		return false;
+
+	return co_realactorheight;
+}
+    VERSION_CONTROL (g_level_cpp, "$Id$")
```
Comment (leximayfield) [inline common/g_level.cpp]: Weird spacing here?

File: common/sc_man.cpp [hunk 4]
```diff
@@ -448,6 +448,31 @@ BOOL SC_GetFloat (void)
 	}
 }
 
+const bool SC_CheckFloat()
```
Comment (leximayfield) [inline common/sc_man.cpp]: You shouldn't return a `const` value type - the `const` is useless.

**Review by leximayfield — CHANGES_REQUESTED** [review]
Just some `const` changes.

**Comment by Ch0wW** [comment]
Latest commit should take care of these small errors.

**Review by leximayfield — APPROVED** [review]
LGTM

**Pull request merged**